classes/user/Repository.php

```diff
@@ -210,13 +212,14 @@ public function getAccessibleWorkflowStages(int $userId, int $contextId, Submiss
             if (array_key_exists($contextId, $userRoles)) {
                 $contextRoles = $userRoles[$contextId];
 
-                foreach ($contextRoles as $contextRole) { /** @var Role $userRole */
+                foreach ($contextRoles as $contextRole) { /** @var Role $contextRole */
```

Comment:
The same comment I left for the other type hint, to fix on the origin, then other near-to-deprecation code will benefit from it.